deploy/helm/sumologic/values.yaml

```
@@ -432,6 +432,8 @@ fluentd:
       ## A regular expression for pods.
       ## Matching pods will be excluded from Sumo. The logs will still be sent to FluentD.
       excludePodRegex: ""
+      perContainerAnnotationsEnabled: false
+      perContainerAnnotationPrefixes: []
```

To me this should be

* a separate section (hence the new line in the suggestion)
* it should have a comment explaining what those setting do

```suggestion
      excludePodRegex: ""

      perContainerAnnotationsEnabled: false
      perContainerAnnotationPrefixes: []
```

---

Thanks! Done.